app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,44 @@
 class BookingsController < ApplicationController
+  before_action :set_offer, only: %i[new create]
+
+  def edit
+  end
+
+  def update
+    if @booking.update(params[:booking])
+      redirect_to bookings_path(@booking), notice: "Successfuly updated booking." #Again with the redirecting
```

Comment:
Yes, I think once we add the overview view/route, then we can update to redirect to that page